ksonnet-gen/kubeversion/data.go

```diff
@@ -68,6 +68,71 @@ var versions = map[string]versionData{
 			"APIVersions":                      "apiVersions",
 			"ServerAddressByClientCIDR":        "serverAddressByClientCidr",
 		},
+		constructorSpecs: map[string][]CustomConstructorSpec{
+			"io.k8s.kubernetes.pkg.api.v1.Container": []CustomConstructorSpec{
+				newConstructor("new", newParam("name"), newParam("image")),
```

Comment:
I would be inclined to start turning things like "new" and "image" into constants